safe_transaction_service/history/management/commands/check_index_problems.py

```diff
@@ -13,57 +11,37 @@
 class Command(BaseCommand):
     help = "Check nonce calculated by the indexer is the same that blockchain nonce"
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.nonce_fn = get_safe_V1_3_0_contract(
-            EthereumClientProvider().w3, address=NULL_ADDRESS
-        ).functions.nonce()
-
     def add_arguments(self, parser):
         parser.add_argument(
             "--fix", help="Fix nonce problems", action="store_true", default=False
         )
 
-    def build_nonce_payload(self, addresses: Iterable[str]) -> Iterable[Dict[str, Any]]:
-        """
-        It looks like web3 takes time generating contract functions, so I do it this way
-        :param addresses:
-        :return:
-        """
-        contract_function = self.nonce_fn
-
-        payloads = []
-        data = contract_function.build_transaction({"gas": 0, "gasPrice": 0})["data"]
-        output_type = [output["type"] for output in contract_function.abi["outputs"]]
-        fn_name = (contract_function.fn_name,)  # For debugging purposes
-        for address in addresses:
-            payload = {
-                "to": address,
-                "data": data,
-                "output_type": output_type,
-                "fn_name": fn_name,
-            }
-            payloads.append(payload)
-        return payloads
+    def get_nonce_fn(self, ethereum_client: EthereumClient):
+        return get_safe_V1_3_0_contract(
+            ethereum_client.w3, address=NULL_ADDRESS
+        ).functions.nonce()
 
     def handle(self, *args, **options):
         fix = options["fix"]
 
         queryset = SafeLastStatus.objects.all()
         count = queryset.count()
-        batch = 100
-        ethereum_client = EthereumClientProvider()
+        batch = 1000
```

Comment:
The batch changed from 100 to 1000. Was this intended?

Reply:
Yes, I was testing and it can safetly increased 👍🏻 . I will edit the PR to address this